scripts/feature_tests/feature_test_vars.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
 
 export NUM_OF_MASTER_REPLICAS="3"
-export NUM_OF_WORKER_REPLICAS="1"
\ No newline at end of file
+export NUM_OF_WORKER_REPLICAS="1"
+export KUBERNETES_VERSION="v1.21.2"
```

Comment:
Does this mean that v1.21.2 becomes the Kubernetes version for all feature tests (instead of 1.22.0)?

Reply:
Yes , thats gonna change pretty soon, once this PR lands and  after a few days of CI sanity I ll do changes for 1.22.0 -> 1.22.1